### PR TITLE
Add OTA terms to Glossary

### PR DESCRIPTION
**Pull request by kprosise**
Added terms gathered by OTA team.

QA Steps: Checked rendered HTML, no issues found. Ran linter and spellcheck.

This commit addresses FFTK-3010

# PR Template and Checklist

Please complete as much as possible to speed up the reviewing process.

Readiness and adding reviewers as appropriate is required.

All PRs should be reviewed by a technical writer/documentation team and a peer.
If effecting customers—which is a majority of content changes—a member of Customer Success must also review.

## Readiness

* [x] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Checklist

* [x] Run spelling and grammar check, preferably with linter.
* [x] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [x] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [x] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [x] View HTML in a browser to check rendering.
* [x] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [x] follow best practices for commits.
  * [x] Descriptive title written in the imperative.
  * [x] Include brief overview of QA steps taken.
  * [x] Mention any related issues numbers.
  * [x] End message with sign off/DCO line (`-s, --signoff`).
  * [x] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [x] Squash commits if needed.
* [x] Request PR review by a technical writer and at least one peer.

### CODE REVIEW

**Comment by doanac** [comment]
Docs for 2a8b46b42416b3ef2b033c0c84d15e70ae6c4b40 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2652/docs/artifacts/html/index.html

File: source/glossary/index.rst [hunk 1]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+      * :ref:`Production Targets Reference Manual, Wave <ref-production-targets>`
+   
+   Wave Rollout
+      An action of rolling out an OTA update associated with a Wave to a subset of Production Devices.
+
+   Wave Tag
+      A tag designating Production Devices to which a given Wave is being provisioned.
+
+   CI Targets
+      TUF Targets created during the CI builds and delivered to Non-Production Devices during an OTA update.
```
Comment (kprosise) [inline source/glossary/index.rst:335]:
```suggestion
      TUF Targets created during the CI builds and delivered to non-production devices during an OTA update.
```

File: source/glossary/index.rst [hunk 2]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+      The FoundriesFactory method for adding a specific CI Targets version to Production Targets.
+      Provisions it to Production Devices in a controlled way.
+
+      * :ref:`Production Targets Reference Manual, Wave <ref-production-targets>`
+   
+   Wave Rollout
+      An action of rolling out an OTA update associated with a Wave to a subset of Production Devices.
+
+   Wave Tag
+      A tag designating Production Devices to which a given Wave is being provisioned.
```
Comment (kprosise) [inline source/glossary/index.rst:332]:
```suggestion
      A tag designating production devices to which a given Wave is being provisioned.
```

File: source/glossary/index.rst [hunk 3]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+      * :ref:`Custom Sota Client User Guide, Custom Update Agent <ug-custom-sota-client>`
+   
+   Wave
+      The FoundriesFactory method for adding a specific CI Targets version to Production Targets.
+      Provisions it to Production Devices in a controlled way.
+
+      * :ref:`Production Targets Reference Manual, Wave <ref-production-targets>`
+   
+   Wave Rollout
+      An action of rolling out an OTA update associated with a Wave to a subset of Production Devices.
```
Comment (kprosise) [inline source/glossary/index.rst:329]:
```suggestion
      An action of rolling out an OTA update associated with a Wave to a subset of production devices.
```

File: source/glossary/index.rst [hunk 4]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+      * :ref:`Reference Manual, Static Deltas <ref-static-deltas>`
+
+   Update Agent
+      Software that runs on a device and performs OTA updates.
+
+      * :ref:`Custom Sota Client User Guide, Custom Update Agent <ug-custom-sota-client>`
+   
+   Wave
+      The FoundriesFactory method for adding a specific CI Targets version to Production Targets.
+      Provisions it to Production Devices in a controlled way.
```
Comment (kprosise) [inline source/glossary/index.rst:324]:
```suggestion
      The FoundriesFactory method for adding a specific CI Targets version to production Targets.
      Provisions it to production devices in a controlled way.
```

File: source/glossary/index.rst [hunk 5]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+      * :ref:`Fioctl User Guide, OSTree Hash <ug-fioctl>`
+
+   Production Device
+      A device with a flag in its certificate which enables it to receive production updates.
+
+      * :ref:`Factory Registration Reference Manual, Registering Proudction Devices by Default <ref-factory-registration-ref>`
+      * :ref:`Reference Manual, Production Targets for Production Devices <ref-production-targets>`
+
+   Production Targets
+      :term:`TUF` Targets delivered to Production Devices during an :term:`OTA Update`.
```
Comment (kprosise) [inline source/glossary/index.rst:291]:
```suggestion
      :term:`TUF` Targets delivered to production devices during an :term:`OTA Update`.
```

File: source/glossary/index.rst [hunk 6]
```diff
@@ -236,3 +236,102 @@ Terminology
...
+
+     * :ref:`Getting Started, Registering Your Device <gs-register>`
+     * :ref:`Device Gateway PKI User Guide, Online Device Certificate Using lmp-device-register <ref-device-gateway-pki-details>`
+     * :ref:`Restorable Apps Reference Manual, extending list of Restorable Apps  Using lmp-device-register <ug-restorable-apps>`
+
+   OTA Update
+   Over-The-Air Update
+   OTA
+      Updating firmware and software for a system/device remotely.
+      The update on a device is triggered remotely and the data fetched from the OTA service via Internet.
```
Comment (kprosise) [inline source/glossary/index.rst:271]:
```suggestion
      The update on a device is triggered remotely and the data fetched from the OTA service via internet.
```

**Review by kprosise — COMMENTED** [review]
Capitalization suggestions

**Comment by doanac** [comment]
Docs for cf4e1ebd7aaa99ccdec231b7d2fdd90127565ee5 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2653/docs/artifacts/html/index.html

**Comment by doanac** [comment]
Docs for 274b5cc0154f8c619c5aa06dea6b546ce7b369a2 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/2654/docs/artifacts/html/index.html

**Pull request merged**